types/postlight__mercury-parser/tsconfig.json

```diff
@@ -15,7 +15,8 @@
         ],
         "types": [],
         "noEmit": true,
-        "forceConsistentCasingInFileNames": true
+        "forceConsistentCasingInFileNames": true,
+        "esModuleInterop": true
```

Comment:
nitpick: I would say remove the `esModuleInterop` flag and instead change the test to `import * as Mercury from 'postlight__mercury-parser';`.

Just to be clear, both imports are correct. I prefer this one just for correctness to avoid enabling flags.

Reply:
Done.